i-VRESSE/haddock3-webapp
# The node buttons render incorrectly

#  The node buttons render incorrectly after switching between pages

Closes #78 

This should fix issue with workflow buttons after switching between app pages.

## Changes made
- Upgrade to core v2 (to address #78)
- Extended builder styles:
   - added bootstrap styles for custom-select
   - added bootstrap styles for popover
   - added bootstrap styles for missing `col-x` classes
   - removed background color for input file type
   - added specific button formatting for new core v2 classes
- Moved css import from builder.tsx to Form.client.tsx imports and removed styles.ts import of table and react toastify styles (these styles seem to be already included - not sure where/how).
- Added gap between buttons
- Removed some div elements
- Defined max/min width of builder page between 800-1600px

## How to test

- Run app and switch between pages, confirm that initial issue #78 is solved
- Confirm that select elements are styled
- Confirm that tables have working popover
- Change theme light/dark and confirm that styles of select and popover are updated
